resteasy-core-spi/src/main/java/org/jboss/resteasy/resteasy_jaxrs/i18n/Messages.java

```diff
@@ -652,9 +652,8 @@
    @Message(id = BASE + 865, value = "Unable to determine base class from Type")
    String unableToDetermineBaseClass();
 
-   @Message(id = BASE
-         + 870, value = "Unable to extract parameter from http request: {0} value is \''{1}\'' for {2}", format = Format.MESSAGE_FORMAT)
-   String unableToExtractParameter(String paramSignature, String strVal, AccessibleObject target);
+   @Message(id = 3870, value = "Unable to extract parameter from http request: %s value is '%s'")
```

Comment:
Did you change BASE+870 into 3870 on purpose?

Reply:
Yes, but I can revert it. IMO it makes it difficult to find messages when the ID is added. When I tried to find "3870" with grep or my IDE it didn't find anything.

I can change it back for sure though. I was going to make a separate proposal to change all id's :) Email first, not just change it all.

Reply:
Yeah, I recognize that there's a little problem searching for a message. I usually grep on a piece of the message itself, which I think is pretty effective. I think that using the BASE is somewhat preferable in that it eliminates a minor (probably "very minor") source of an error. In any case, I think it's better to have everything consistent, so I would suggest reverting the change and then discussing the whole issue with the team. I'm about a -0.2 on it, but maybe there's a general sense going the other way.

Reply:
Okay. I'll change this part back. We have a more in depth discussion elsewhere as this was not the place to make a change like this :)